influxdb_client/client/query_api.py

```diff
@@ -146,10 +156,53 @@ def query_data_frame_stream(self, query: str, org=None, data_frame_index: List[s
 
     # private helper for c
     @staticmethod
-    def _create_query(query, dialect=default_dialect):
-        created = Query(query=query, dialect=dialect)
+    def _create_query(query, dialect=default_dialect, params: dict = None):
+        created = Query(query=query, dialect=dialect, extern=QueryApi._build_flux_ast(params))
         return created
 
+    @staticmethod
+    def _params_to_extern_ast(params: dict) -> List['OptionStatement']:
+
+        statements = []
+        for key, value in params.items():
+
+            if isinstance(value, bool):
+                literal = BooleanLiteral("BooleanLiteral", value)
+            elif isinstance(value, int):
+                literal = IntegerLiteral("IntegerLiteral", str(value))
+            elif isinstance(value, float):
+                literal = FloatLiteral("FloatLiteral", value)
+            elif isinstance(value, datetime):
+                if not value.tzinfo:
+                    value = UTC.localize(value)
+                else:
+                    value = value.astimezone(UTC)
```

Comment:
Please, move this into `date_utils.py` together with https://github.com/influxdata/influxdb-client-python/blob/e58467651b8f12b911961968d9fbf468ebbcf116/influxdb_client/client/write/point.py#L207

It will be useful for user if they want to handle `datetime` different way - see #73